google-cloud-tools-plugin/src/com/google/gct/idea/appengine/cloud/DoManagedVmDeployment.java

```diff
@@ -122,6 +122,7 @@ public void run() {
     commandLine.withWorkDirectory(stagingDirectory);
     consoleLogLn(loggingHandler, "Working directory set to: " + stagingDirectory.getAbsolutePath());
     commandLine.withParentEnvironmentType(ParentEnvironmentType.CONSOLE);
+    commandLine.getEnvironment().put("CLOUDSDK_METRICS_ENVIRONMENT", "gcloud-intellij");
```

Comment:
Yes, I did a bunch of refactoring to these deployment "actions" in my other PR - #541 (still in progress). I can extract that out too.